src/index.d.ts

```diff
@@ -388,6 +388,28 @@ export interface BadgeProps {
  */
 export class Badge extends React.Component<BadgeProps> {}
 
+/**
+ * withBadge Higher-Order Component
+ * @param value 
+ * @param options 
+ */
+export function withBadge(
+  /**
+   * Text value to be displayed by badge
+   */
+  value: React.ReactNode | (() => React.ReactNode),
+  /**
+   * Options to configure the badge
+   */
+  options: {,
```

Comment:
Lastly this should be optional, so `options?: `

Reply:
Updated!